cdc/processor/pipeline/table.go

```diff
@@ -100,8 +100,7 @@ func (t *tablePipelineImpl) UpdateBarrierTs(ts model.Ts) {
 // AsyncStop tells the pipeline to stop, and returns true is the pipeline is already stopped.
 func (t *tablePipelineImpl) AsyncStop(targetTs model.Ts) bool {
 	err := t.p.SendToFirstNode(pipeline.CommandMessage(&pipeline.Command{
-		Tp:        pipeline.CommandTypeStopAtTs,
-		StoppedTs: targetTs,
+		Tp: pipeline.CommandTypeStopAtTs,
```

Comment:
```suggestion
		Tp: pipeline.CommandTypeStopAtTs,
```

Cloud you rename `pipeline.CommandTypeStopAtTs` to `pipeline.CommandTypeStop` ?